src/main/resources/templates/db_scripts/schema.json

```diff
@@ -96,6 +96,11 @@
         }
       ],
       "customSnippetPath" : "document_table.sql"
+    },
+    {
+      "tableName": "batch_vouchers",
+      "fromModuleVersion": "mod-invoice-storage-3.1.0",
```

Comment:
Please, use here mod-invoice-storage-4.0.0 because the next release will be increased to major version due to breaking changes.